pkg/slack/slack.go

```diff
@@ -62,22 +66,33 @@ func (b *Bot) Start() {
 			}
 
 			// Check if message posted in a channel
-			_, err := api.GetChannelInfo(ev.Channel)
+			channel, err := api.GetChannelInfo(ev.Channel)
 			if err == nil {
 				// Message posted in a channel
 				// Serve only if starts with mention
 				if !strings.HasPrefix(ev.Text, "<@"+botID+"> ") {
 					continue
 				}
+				// if checkChannel is true
+				// Serve only if current channel is in config
+				if b.CheckChannel && (b.ChannelName != channel.Name) {
+					continue
+				}
+
 			} else {
 				// Check if message posted in a group
-				_, err := api.GetGroupInfo(ev.Channel)
+				group, err := api.GetGroupInfo(ev.Channel)
 				if err == nil {
 					// Message posted in a group
 					// Serve only if starts with mention
 					if !strings.HasPrefix(ev.Text, "<@"+botID+"> ") {
 						continue
 					}
+					// if checkChannel is true
+					// Serve only if current group is in config
+					if b.CheckChannel && (b.ChannelName != group.Name) {
```

Comment:
@gimmetm can you please remove this check? We should be able to execute BotKube commands from group chats.

Reply:
@PrasadG193  This check is for private channel. (https://api.slack.com/methods/groups.info)
group chat info might be in conversation.info or else.